docs/source/user-guide/nlp/supported_tasks.rst

```diff
@@ -22,28 +22,31 @@ Deepchecks currently supports two NLP task types:
   sample. Examples for such tasks are: Sentiment Analysis, Topic Extraction, harmful content detection and more.
 * **Token Classification**: Token Classification is any NLP task in which each word (or to be more accurate, token) in
   the text sample is assigned a class of its own. In many cases most tokens will belong to a "background" class,
-  allowing the model to focus on the interesting tokens. Examples for such tasks are: Named Entity Recognition, Part-of-speech annotation (in which all tokens have a non-background class).
+  allowing the model to focus on the interesting tokens. Examples for such tasks are: Named Entity Recognition,
+  Part-of-speech annotation (in which all tokens have a non-background class).
```

Comment:
Would consider making the examples here and in previous stand out more by bullets.
Note - my syntax isn't full cause couldn't do the suggestion properly on all lines (and better to verify with the bolilerplate anyway)

Or maybe simpler than bullets - links?
Any way that would make it to jump to the eye of the reader so they can easily see the actual/specific types of tasks that are supported

```suggestion
  allowing the model to focus on the interesting tokens.
  | Examples for such tasks are: 
  | - Named Entity Recognition
  | - Part-of-speech annotation (in which all tokens have a non-background class).
```